docs/source/conf.py

```diff
@@ -126,10 +123,8 @@ def linkcode_resolve(domain, info):
             source, lineno = inspect.getsourcelines(obj)
         linespec = f"#L{lineno}-L{lineno + len(source) - 1}"
     except TypeError:
-            linespec = ""
+        linespec = ""
     return (
         f"https://github.com/snowflakedb/snowpark-python/blob/"
-        f"v{release}/{os.path.relpath(fn, start=os.pardir)}{linespec}"
+        f"release-v{release}/{os.path.relpath(fn, start=os.pardir)}{linespec}"
```

Comment:
Why did we add the `release` prefix here?

Reply:
I think snowpandas was working with an older version of this file - looks like the "release" part was removed a month ago, I can get rid of it to keep up with the new change.

Reply:
We should stick with the Snowpark Python version, since I don't think this link ever was used by Snowpark pandas in the first place.